share/instruments/instruments.xml

```diff
@@ -6671,7 +6671,6 @@
                   <Channel>
                         <controller ctrl="0" value="1"/>
                         <program value="48"/> <!--Orchestra Kit-->
-                        <controller ctrl="10" value="60"/>
```

Comment:
This might be needed

Reply:
This one is annoying because the others all have zero or one controller. Adding a second one makes things much more complicated. Could we just use `<controller ctrl="10" value="60"/>` and get rid of `<controller ctrl="0" value="1"/>` or are both needed?

Reply:
Not sure.

Reply:
As discussed on Discord, that like came in via  15631d3, by Werner some 9 years ago, `ctrl="10"` is volume, as per event.h. I can't spot any differences between a Cymbal with and without that line in sound/volume, so it seems OK to just get rid if it. The other line, `<controller ctrl="0" value="1"/>` is needed though, for every percussion instrument.

Reply:
It's the cymbal's pan that has been set to 60, not volume (see [General MIDI: Controller Events](https://en.wikipedia.org/wiki/General_MIDI#Controller_events)). The default pan is 64 (centred), so 60 is not noticeably different. I think this is a relic from an effort (later abandoned and partially reverted) to add pan to all the instruments to arrange them like a symphony orchestra. It was a nice idea, but poorly executed because it would also affect smaller ensemble scores, potentially leading to all instruments playing in one ear only. So it should be safe to remove (desirable even).

Reply:
Ah, I see, 7 is volume, 10 is pan, even more reason to get rid of it